symr/benchmark.py

```diff
@@ -117,15 +125,15 @@ def evaluate(**kwargs):
         sr_methods = [sr_methods]
 
     for method in sr_methods:
-        for expr_index, expression in enumerate(expressions):
```

Comment:
This was an extra change. Instead of looping through each trial first, I swapped the trial and expression loops to run breadth-first benchmarks.